File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/edne-registrar.request.ts
import { EdneListarParamEstructuraResponse } from "../response/edne-listar-param-estructura.response";
import { EdneRegistrarDetUsuarioRequest } from "./edne-registrar-det-usuario.request";

export class EdneRegistrarRequest {
    idEstructura: number;
    nomArchivo: string;
    fecImportacion: Date;
    fecPeriodo: Date;
    idtEstado: string;
    // listaUsuarioIdentificado: EdneRegistrarDetUsuarioRequest[];
    // listaUsuarioNoIdentificado: EdneRegistrarDetUsuarioRequest[];
    // queryUsuario: string;
    // queryUsuarioDetalle: string;
    // queryUsuarioIngreso: string;
    // queryAgenteExterno: string;
    // queryUsuarioActividad: string;
    // queryUsuarioSituacion: string;
    listaParametro: EdneListarParamEstructuraResponse[];
    listaUsuario: any[];
    
    idUsuarioCrea: number;
}

File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/EdneRegistrarRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.Date;
import java.util.List;
import java.util.Map;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
import pe.gob.mimp.reni.dto.response.EdneListarParamEstructuraResponse;

@Getter
@Setter
@ToString
public class EdneRegistrarRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idEstructura;
	private String nomArchivo;
	private Date fecImportacion;
	private Date fecPeriodo;
	private String idtEstado;
	private List<EdneListarParamEstructuraResponse> listaParametro;
	private List<Map<String, Object>> listaUsuario;
//	private List<EdneRegistrarDetUsuarioRequest> listaUsuarioIdentificado;
//	private List<EdneRegistrarDetUsuarioRequest> listaUsuarioNoIdentificado;
//	private String queryUsuario;
//	private String queryUsuarioDetalle;
//	private String queryUsuarioIngreso;
//	private String queryAgenteExterno;
//	private String queryUsuarioActividad;
//	private String queryUsuarioSituacion;
	private Long idUsuarioCrea;
}
